src/soilbiogeochem/SoilBiogeochemStateType.F90

```diff
@@ -34,7 +34,6 @@ module SoilBiogeochemStateType
      real(r8) , pointer :: fpi_col                     (:)     ! (no units) fraction of potential immobilization 
      real(r8),  pointer :: fpg_col                     (:)     ! (no units) fraction of potential gpp 
      real(r8) , pointer :: rf_decomp_cascade_col       (:,:,:) ! (frac) respired fraction in decomposition step 
-     real(r8) , pointer :: pathfrac_decomp_cascade_col (:,:,:) ! (frac) what fraction of C leaving a given pool passes through a given transition 
```

Comment:
why is this being removed from this module?  
Seeing this code it also makes me wonder if we need to:
- write out `fpi_col` (N limitation of decomp), or if it's brought into `decomp_k`?
- write out `rf_decomp_cascade_col`, or is this brought into `pathfrac` somewhere?

Reply:
`pathfrac` got moved from here to `SoilBiogeochemCarbonFluxType` for reasons outlined in [this comment](https://github.com/ESCOMP/CTSM/pull/1457/files#r686392731)

FPI is on the master list as an inactive field.

`rf_decomp_cascade` is not. Let me know if I should add it.

Reply:
Let's go ahead and write out fpi and rf_decomp_cascade to history.  I think we'll want these for the precondiotnal